backend.native/tests/build.gradle

```diff
@@ -5002,7 +5078,8 @@ private void configureStdlibTest(KonanGTest task, boolean inWorker) {
     task.useFilter = false
     task.testLogger = KonanTest.Logger.GTEST
     task.finalizedBy("resultsTask")
-    task.enabled = (project.testTarget != 'wasm32')  // Uses exceptions
+    task.enabled = (project.testTarget != 'wasm32') && // Uses exceptions
+        !ext.isExperimentalMM  // Experimental MM doesn't support multiple mutators yet.
```

Comment:
Doesn't this test run in a single thread if `inWorker  == false`?

Reply:
Good point, thanks! Triggered https://buildserver.labs.intellij.net/buildConfiguration/Kotlin_KotlinNative_Development_KotlinNativeMacSanityNewMm/105992740